src/models/util/LoadShow.ts

```diff
@@ -4,13 +4,6 @@ import { ParseCalChart } from "./ParseCalChart";
 import { ParseCalChart31 } from "./ParseCalChart31";
 import { ParseCalChart34 } from "./ParseCalChart34";
 
-// Importing calchart4 is not yet supported
-/* eslint-disable @typescript-eslint/no-unused-vars */
-function IsCalChart4(buffer: ArrayBuffer): ParseCalChart | null {
```

Comment:
I removed this part of the parsers because I was having a very hard time in Typescript converting from an ArrayBuffer to a string that json could parse... Ultimately it became easier to just check if the filename ended in a .shw4 and check based on that.